src/components/layout/nav/components/logo-section.tsx

```diff
@@ -53,22 +53,25 @@ const query = graphql`
 const LogoSection = ({ is_ppc_redirect, base, hide_get_trading }: LogoSectionProps) => {
     const data = useStaticQuery(query)
     const to = is_ppc_redirect ? '/landing' : base || '/'
-
+    const { hide_branding } = React.useContext(DerivStore)
     return (
         <Wrapper>
             <LogoLink to={to} aria-label="Home">
-                <QueryImage
-                    data={data['deriv']}
-                    alt="deriv logo"
-                    max_width="16.4rem"
-                    width="100%"
-                    height="auto"
-                />
+                {!hide_branding && (
+                    <QueryImage
+                        data={data['deriv']}
+                        alt="deriv logo"
+                        max_width="16.4rem"
+                        width="100%"
+                        height="auto"
+                    />
+                )}
             </LogoLink>
+
             {!hide_get_trading && (
                 <>
-                    <Line />
-                    <img src={GetTrading} alt="get trading" />
+                    {!hide_branding && <Line />}
+                    {!hide_branding && <img src={GetTrading} alt="get trading" />}
                 </>
             )}
```

Comment:
This can be wrapped in one condition

```
 {!hide_get_trading && !hide_branding && (
                <>
                   <Line />
                   <img src={GetTrading} alt="get trading" />
                </>
            )}
```

Reply:
okay@prince-deriv ..Thank you